src/test/java/LoginTests.java

```diff
@@ -8,35 +9,37 @@
 public class LoginTests extends BaseTest {
 
     //    @Test(enabled = true, priority = 0, description = "LoginEmptyEmailPasswordTest")
-    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
-    public static void loginEmptyEmailPasswordTest (String email, String password) {
-
-        login(email, password);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
+//    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
+//    public static void loginEmptyEmailPasswordTest (String email, String password) {
+//        //login(email, password);
+//        LoginPage loginPage = new LoginPage(getDriver());
+//        loginPage.provideEmail("demo@class.com");
+//        loginPage.providePassword("");
+//        loginPage.clickSubmitBtn();
+//        //wait.
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.